src/TestProblem/Hydro/CCSN/dtSolver_CCSN.cpp

```diff
@@ -148,3 +151,38 @@ double Mis_GetTimeStep_Lightbulb( const int lv, const double dTime_dt )
    return dt_LB;
 
 } // FUNCTION : Mis_GetTimeStep_Lightbulb
+
+
+
+//-------------------------------------------------------------------------------------------------------
+// Function    :  Mis_GetTimeStep_CoreCollapse
+// Description :  Set the time-step to CCSN_CC_Red_DT during the core collape once the central 
+//                density reaches CCSN_CC_CentralDensFac (g/cm^3) to get a more accurate bounce time
+//
+// Note        :  1. This function should be applied to both physical and comoving coordinates and always
+//                   return the evolution time-step (dt) actually used in various solvers
+//                   --> Physical coordinates : dt = physical time interval
+//                       Comoving coordinates : dt = delta(scale_factor) / ( Hubble_parameter*scale_factor^3 )
+//                   --> We convert dt back to the physical time interval, which equals "delta(scale_factor)"
+//                       in the comoving coordinates, in Mis_GetTimeStep()
+//                2. Invoked by Mis_GetTimeStep() using the function pointer "Mis_GetTimeStep_User_Ptr",
+//                   which must be set by a test problem initializer
+//                3. Enabled by the runtime option "OPT__DT_USER"
+//
+// Parameter   :  lv       : Target refinement level
+//                dTime_dt : dTime/dt (== 1.0 if COMOVING is off)
+//
+// Return      :  dt
+//-------------------------------------------------------------------------------------------------------
+double Mis_GetTimeStep_CoreCollapse( const int lv, const double dTime_dt )
+{
+
+   double dt = HUGE_NUMBER;
+   
+   const double CentralDens = CCSN_CentralDens / UNIT_D;
+   
```

Comment:
Remove the trailing space in lines 181 and 183

Reply:
and line 159

Reply:
Removed trailing spaces